components/user_settings/advanced/user_settings_advanced.jsx

```diff
@@ -320,6 +321,16 @@ export default class AdvancedSettingsDisplay extends React.PureComponent {
     render() {
         const serverError = this.state.serverError || null;
         let ctrlSendSection;
+        let fmIdForUserSettingsAdvSendDesc = t('user.settings.advance.sendDesc');
+        let fmIdForUserSettingsAdvSendTitle = t('user.settings.advance.sendTitle');
+        let defaultMessageSendDesc = 'When enabled, CTRL + ENTER will send the message and ENTER inserts a new line.';
+        let defaultMessageSendTitle = 'Send messages on CTRL+ENTER';
+        if (Utils.isMac()) {
+            fmIdForUserSettingsAdvSendDesc = t('user.settings.advance.sendDesc.mac');
+            fmIdForUserSettingsAdvSendTitle = t('user.settings.advance.sendTitle.mac');
+            defaultMessageSendDesc = 'When enabled, CMD + ENTER will send the message and ENTER inserts a new line.';
+            defaultMessageSendTitle = 'Send messages on CMD+ENTER';
+        }
```

Comment:
Please extract into a function so we can get values by:
```javascript
const {ctrlSendTitle, ctrlSendDesc} = this.getCtrlSendText();
```
The paired translation strings above are not easy to follow, could you make it something like?
```javascript
const description = {
    default: {
        id: t('user.settings.advance.sendDesc'),
        defaultMessage: 'When enabled, CTRL + ENTER will send the message and ENTER inserts a new line.',
    },
    mac: {
        id: t('user.settings.advance.sendDesc.mac'),
        defaultMessage: 'When enabled, CMD + ENTER will send the message and ENTER inserts a new line.',
    },
};
```
Then return value as `ctrlSendDesc: description.mac` if Mac or `.default` otherwise.  Same case for title.

Reply:
Oh, I just read now the previous comment fro HH.  I'm good then with any approach.

Reply:
Oh yeah, that also works. That would help keep it a bit cleaner as well

Reply:
I like this refactor